core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/PositionalParametersStoredProcedureCallTest.java

```diff
@@ -46,7 +45,7 @@
  * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
  */
 @SkipByGridDialect(
-		value = { HASHMAP, INFINISPAN, INFINISPAN_REMOTE, NEO4J_EMBEDDED, NEO4J_REMOTE },
+		value = {HASHMAP, INFINISPAN, INFINISPAN_REMOTE, NEO4J_REMOTE},
 		comment = "These dialects not support stored procedures with positional parameters")
 @TestForIssue(jiraKey = { "OGM-359" })
 public class PositionalParametersStoredProcedureCallTest extends OgmJpaTestCase {
```

Comment:
What about `NamedParametersStoredProcedureCallTest`?

Reply:
I've added tests for named params